webrender/src/prim_store.rs

```diff
@@ -735,14 +738,22 @@ impl PrimitiveStore {
                 PrimitiveKind::RadialGradient=> {}
                 PrimitiveKind::TextRun => {
                     let text = &mut self.cpu_text_runs[metadata.cpu_prim_index.0];
+                    let sc_index = text.stackingcontext_index;
```

Comment:
Let's make the prims_to_resolve field in PrimitiveStore a ```Vec<(StackingContextIndex, PrimitiveIndex)>```. Then the stacking context index is available for each primitive, and we don't need to store it in the primitive data itself.

This is still not ideal - since it means a hash lookup in the aux lists for each text run, rather than once per stacking context. But we can fix that later, as a follow up.